### DISCUSSION TITLE
The info at the bottom of the home page cannot be selected for copying

### DISCUSSION THREAD

**Discussion by Proaxel** [wstyres/Zebra#1374] [Ideas]
**Describe the bug**
Both Cydia and Zebra home pages list the same info at the bottom of their home pages, which are the device name, UDID, iOS version, etc. However, Cydia allows you to select the text by touching holding said info, while nothing happens if you do the same thing in Zebra.

**To Reproduce**
Tap and hold the info at the bottom of home screen. In Cydia this selects the info, while in Zebra this doesn't happen.

**Expected behavior**
Cydia allows you to select the info like it's any other text and do the normal actions for selecting text like copying to clipboard. The same thing should happen in Zebra.

**Screenshot**
This is how it works in Cydia, and how it should work in Zebra:
![FCDF406E-1D60-467F-9FDC-B85A604EE1E9](https://user-images.githubusercontent.com/9545347/83686472-1b064400-a59f-11ea-95b5-1fb3c1924e99.jpeg)

**Device Information (please complete the following information):**

***Please do not use the word "latest" for Zebra Version because it's inconvenient for us to know the actual version you are talking about - harder to pinpoint the issue.***
 - iOS Version [e.g. 12.1.2 or 10.3.3]: <13.5>
 - Zebra Version [e.g. 1.0\~beta1 or beta1]: <1.1.10>
 - Device Type [e.g. iPhone or iPad]: <iPhone 6s+>
 - Jailbreak Type [e.g. unc0ver, Chimera, checkra1n, etc.]: checkra1n

**Comment by mwoolweaver** [comment]
can confirm the same behaviour on Zebra 1.2~beta1b.

checkra1n 0.10.2, iOS 13.5, iPhone 8 Plus

**Comment by wstyres** [comment]
This isn't a bug, we'll consider it a feature request
